docs/11_view.md

```diff
@@ -0,0 +1,85 @@
+# View Dandisets
+
+## Browse `Dandisets`
+
+When you go the [DANDI Web application](https://dandiarchive.org/) you can click 
+on `PUBLIC DANDISET` to have access to all `Dandisets` currently available 
+in the archive and you can sort them  by name, identifier or date of modification. 
+
+<img
+src="./img/web_browse.png"
+alt="web_browse"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+
+## Search `Dandisets`
+In addition you can search across the `Dandisets` for any text part of the Dandiset metadata record. 
+The text may be about contributor names, modalities, or species.  For example,  `"house mouse"` will
+return a subset of all `Dandisets`, while `"mouse house"` will likely not return any. When unquoted each
+word is used as an `OR`.
+
+<img
+src="./img/web_search.png"
+alt="web_search"
+style="width: 60%; height: auto; display: block; margin-left: auto;  margin-right: auto;"/>
+
+
+When you click on one of the `Dandisets` you can see that the searching phrase can
```

Comment:
When you click on one of the `Dandisets`, you can see that the searching phrase can